### PR TITLE
UI Top menu from code

### PR DESCRIPTION
**Pull request by TheAkashicTraveller**
Made changes to load the top menu from a "UI Controller".

### CODE REVIEW

**Comment by Sommerbo** [comment]
In your PR you have a command summary of vi/vim I don't think it needs to be merged. I might be wrong though.

**Comment by TheAkashicTraveller** [comment]
What exactly are you referring to by that?

File: to fit proposed coming style (-24) [hunk 1]
```diff
@@ -0,0 +1,233 @@
+
+                   SSUUMMMMAARRYY OOFF LLEESSSS CCOOMMMMAANNDDSS
+
+      Commands marked with * may be preceded by a number, _N.
+      Notes in parentheses indicate the behavior if _N is given.
+      A key preceded by a caret indicates the Ctrl key; thus ^K is ctrl-K.
+
```
Comment (henninglive) [inline to fit proposed coming style (-24)]: why havd you added this file?

**Comment by Sommerbo** [comment]
I saw this at the bottom of "Files changed" page 

233  to fit proposed coming style (-24)
@@ -0,0 +1,233 @@
 +
 +                   SSUUMMMMAARRYY OOFF LLEESSSS CCOOMMMMAANNDDSS
 +
 +      Commands marked with * may be preceded by a number, _N.
 +      Notes in parentheses indicate the behavior if _N is given.
 +      A key preceded by a caret indicates the Ctrl key; thus ^K is ctrl-K.
 +
 +  h  H                 Display this help.
 +  q  :q  Q  :Q  ZZ     Exit.
 + ---------------------------------------------------------------------------

**Comment by TheAkashicTraveller** [comment]
I'm not sure why that's there, I thing it came from the git bash text editor. I've removed it from mine do you know how I can update my pull request? Should I just close and resubmit?

**Comment by henninglive** [comment]
Just push to the branch on your fork. The pull request gets updated automatically.

**Comment by TheAkashicTraveller** [comment]
Done

**Comment by TheAkashicTraveller** [comment]
So one of the new commits introduced some conflicts and I have no idea how to fix this.

**Comment by relinu** [comment]
#317 isn't merged yet but you can look at the files changed there is a tutorial

**Comment by TheAkashicTraveller** [comment]
I'm thinking it would be easier to close it, copy the changes I made somewhere, delete my fork and do it again.

**Pull request closed without merging**